Autopilot/AttitudeManager/VN_lib.h

```diff
@@ -47,7 +47,7 @@
 #define VN_BYTE4(word)    ((unsigned char)((word) & 0x000000FF))
 
 /* Bit mask to get the nth byte out of a 32-bit word where n=0 is most significant and n=3 is least significant */
-#define VN_BYTE(word, n)   ((unsigned char)((word & (0x000000FF << (n*8))) >> (n*8)))
+#define VN_BYTE(word, n)   ((unsigned char)((word & (0x000000FF << (n*8))) >> (n*8))) // doesn't work?
```

Comment:
Should work unless its bigger than a byte, then this function wouldn't work.

What's going on here?